soap/src/java/com/zimbra/soap/type/GlobalS3BucketConfiguration.java

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
for soap api, is this class for some request or response?

Reply:
Yes.